src/main/java/com/mentalhealthapi/mentalhealthapi/dto/Disorder.java

```diff
@@ -6,6 +6,6 @@
     private int id;
     private String name;
     private String description;
-    private String dsm5Page;
+    private int diagnosticStatisticalManualPage;
```

Comment:
Renamed to use the whole name of the manual so that users will know what the variable is.

Reply:
Great idea.  

Make names descriptive, which often means making them long.  The time you save in typing an abbreviated name will quickly be lost when you forget what abbreviation you used.